pkg/platform/apigateway.go

```diff
@@ -61,13 +65,43 @@ type APIGatewayScrubber struct {
 	*common.AbstractScrubber
 }
 
-func NewAPIGatewayScrubber(sensitiveFields []*regexp.Regexp, kubeClientSet kubernetes.Interface) *APIGatewayScrubber {
-	abstractScrubber := common.NewAbstractScrubber(sensitiveFields, kubeClientSet)
+func NewAPIGatewayScrubber(parentLogger logger.Logger,
+	sensitiveFields []*regexp.Regexp,
+	kubeClientSet kubernetes.Interface) *APIGatewayScrubber {
+	abstractScrubber := common.NewAbstractScrubber(parentLogger, sensitiveFields, kubeClientSet, common.ReferencePrefix, common.NuclioResourceLabelKeyApiGatewayName, SecretTypeAPIGatewayConfig, func(secret v1.Secret) bool {
+
+		// since we can create 2 secrets for API Gateway with the same selector,
```

Comment:
Why is this?..

Reply:
@TomerShor 1st is with masked data, 2nd is for nginx